_source/_posts/2022-08-15-spring-boot-crud-update.md

```diff
@@ -0,0 +1,1077 @@
+---
+layout: blog_post
+title: "Spring Boot CRUD app with Vue and Quasar"
```

Comment:
We should match the title and the URL of the one we're trying to update as much as possible. We're updating the following post with this one:

https://developer.okta.com/blog/2018/11/20/build-crud-spring-and-vue

You might even want to match the headings. We don't want Google to think too much has changed. You'll also need to update [`_redirects_`] and and add a `redirect_to` in the front matter of the previous post.

```yaml
redirect_to:
  - "/blog/2022/08/18/build-crud-spring-and-vue"
```

Reply:
So, something like "Build a Spring Boot CRUD App with Vue.js and Quasar" OR "Build a Simple CRUD App with Spring Boot, Vue.js, and Quasar" OR can we leave it exactly as is?

Reply:
I think the same title is best. For my Spring Boot SAML post, I did alter it slightly, removing "Okta" from the title.

Reply:
wfm!